.github/workflows/initdb.yml

```diff
@@ -42,12 +42,11 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
 
-      # Build and push KeyCloak custom images for central and shared idp instances
-      - name: 'Build images'
+      - name: 'Build and push initdb Docker image'
         uses: docker/build-push-action@v2
         with:
           context: .
           file: docker/Dockerfile.import
           push: true
-          tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
+          tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:1.0.0
```

Comment:
This workflow will build and tag a container image, right? Hard coding the image tag is not recommended, because you've to adjust each time the workflow when the image will be changed.

Maybe it's worth taking a look into our [Example](https://github.com/catenax-ng/k8s-helm-example/blob/main/.github/workflows/docker-build.yaml). In this case a image will only be tagged with semVer when a tag is pushed to the repo (this will usually happen while creating a GH release for example).